src/components/Modal/Modal.tsx

```diff
@@ -10,6 +10,7 @@ const ModalOverlay = styled.div<{ noBackground?: boolean }>`
   padding: 15px;
   background: rgba(0, 0, 0, 0.4);
   border-radius: 12px;
+  z-index: ${({ isComponentOnTop }) => !!isComponentOnTop && '1'};
```

Comment:
what happens if you don't pass the `isComponentOnTop` property? `zindex: ;`

Reply:
Some UI issues with displaying the AuthChain label on top. But realised with some changes it should work without z-index.